go/service/main.go

```diff
@@ -184,6 +187,17 @@ func (d *Service) writeServiceInfo() error {
 	return rtInfo.WriteFile(path.Join(runtimeDir, "keybased.info"))
 }
 
+func (d *Service) checkTracksEveryHour() {
```

Comment:
Is there a mechanism to rate-limit these checks if they happen more frequently than some lower limit?  You can imagine having 10 windows open all at once, each asking to check once a minute, which means you'll get 10 checks a minute in the simple implementation.